rir/src/compiler/translations/rir_2_pir.h

```diff
@@ -1,37 +1,19 @@
 #ifndef RIR_2_PIR_H
```

Comment:
maybe we should move the rir_2_pir.[h|cpp] as well as the pir/stackmachine.[h|cpp] to a `rir_2_pir_compiler` subfolder?